src/screens/ShareTrip/ShareTrip.tsx

```diff
@@ -0,0 +1,158 @@
+                    );
+                }
+            })
+            .catch((error) => console.error(error)); // TODO - Notify user with error modal
+    };
+
+    const handleSystemShareSheet = async (invitationLink: string) => {
+        try {
+            let shareObject: ShareContent = {
+                title: t("screens.shareTrip.androidShareSheetTitle"),
+                message: invitationLink,
+            };
+            // Only use url to properly display shareable content in iOS share sheet
+            if (Platform.OS === "ios") {
+                shareObject = {
+                    url: invitationLink,
+                };
+            }
+
+            await Share.share({ ...shareObject });
+        } catch (error) {
+            // TODO - Notify user with error modal
+            console.error(error.message);
+        }
+    };
+
+    return (
+        <>
+            <Header
+                placement={"left"}
+                containerStyle={styles.header}
+                leftComponent={<Text h2>{trip.name}</Text>}
+                rightComponent={
+                    <Avatar
+                        rounded
+                        size="medium"
+                        icon={{
+                            name: "umbrella-beach",
+                            type: "font-awesome-5",
+                        }}
+                        containerStyle={{ backgroundColor: "black" }}
+                    /> // TODO - Add proper icon once backend provides one
+                }
+            />
+            <ScrollView style={styles.scrollView}>
+                {trip.description && <Text h4>{trip.description}</Text>}
+                <SvgLogo style={styles.logo} height={150} width={150} />
+                <Button
+                    containerStyle={styles.btnContainer}
+                    buttonStyle={styles.shareBtn}
+                    title={t("screens.shareTrip.share")}
+                    titleStyle={styles.btnTextStyle}
+                    onPress={() => handleShare()}
+                    loading={loading}
+                />
+                <Button
+                    containerStyle={styles.btnContainer}
+                    buttonStyle={styles.planTripBtn}
+                    title={t("screens.shareTrip.planTrip")}
+                    titleStyle={styles.btnTextStyle}
+                    // TODO - Redirect to trip itinerary screen
+                    onPress={() =>
+                        console.log(
+                            "TODO - Missing redirect to trip details screen"
+                        )
+                    }
+                />
+                <Button
+                    containerStyle={{ marginTop: -10 }}
+                    buttonStyle={styles.backToDashboardBtn}
+                    title={t("screens.shareTrip.backToDashboard")}
+                    titleStyle={{ color: "black", fontSize: 20 }}
+                    onPress={() => props.navigation.replace("Dashboard")}
```

Comment:
Nothing you need to fix right now but I think we should start declaring routes only once in an exported enum, so that we don't manually repeat the route name over and over again (`Dashboard`)

Reply:
I've created a new issure reagarding this topic: #31